x-pack/plugins/cloud/server/saved_objects/get_cloud_onboarding_data.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+
+export const getCloudOnboardingData = (savedObjectClient)
```

Comment:
@XavierM you need to import the SO client

Reply:
that's old